app/channels/course_channel.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
```

Comment:
Could we create a separate file for this (ie ApplicationChannel)? Then move all course-related channels to course folder. We can then separate the concerns for app vs course level.. 

This is to allow channel to be used above course level in the future..

Reply:
As discussed, we'll move `ApplicationCableMultitenancyConcern` up to `ApplicationCable::Channel` and move the two channels under `course/monitoring/` to follow the convention in `controllers/`